README.md

```diff
@@ -68,53 +68,48 @@ The idea is to install "Legacy Version" of WhatsApp on you device via adb and us
+For that go to 'WhatsApp settings\Chat Settings\Chat Backup' here take a local backup. Prepare for the worst.               
+After [installation](#installation-and-usage) follow on screen instructions.
 
 ![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
 
-### Prerequisites
+### Prerequisites and preparation
 
 * USB Debugging must be enabled on the target device. Settings -> Developer Options -> (Debugging) USB debugging  
-     If you cannot find Developer Options then please go to: Settings -> About phone/device and tap the Build number multiple times until you're finally declared a developer. 
+  * If you cannot find Developer Options then please go to: Settings -> About phone/device and tap the Build number multiple times until you're finally declared a developer. 
 * Android device with Android 11 or higher with activated 'ADB over network' setting. Settings -> Developer Options -> ADB over network
 * OR an Android device with Android 4 to 10 (i.e. Ice Cream Sandwich, Jelly Bean, KitKat, Lollipop, Marshmallow, Nougat, Oreo, Pie, Q) AND a PC with USB functionality and installed [Android Debug Bridge for PC](https://developer.android.com/studio/releases/platform-tools)
   * Run the following commands in a terminal opened in the 'platform-tools' folder. Authorize on your phone when prompted:
-  * `adb devices`
-  * `adb tcpip 5555`
+   * `adb devices`
+   * `adb tcpip 5555`
 * Termux : Install from [Play Store](https://play.google.com/store/apps/details?id=com.termux) or [F-Droid](https://f-droid.org/packages/com.termux)
 
-Run the following commands in the Termux app:
-* Python 3.x : `pkg install python -y`
-* Proot : To make Java work `pkg install proot -y && proot login`
-* [Java](https://github.com/Hax4us/java/) : `pkg update && pkg install wget -y && wget https://raw.githubusercontent.com/Hax4us/java/master/installjava && sh installjava`
-* [ADB (Android Debug Bridge)](https://github.com/MasterDevX/Termux-ADB) Drivers : `pkg update && pkg install wget && wget https://github.com/MasterDevX/Termux-ADB/raw/master/InstallTools.sh && bash InstallTools.sh`
-* Git : `pkg install git`
-
-
-
 ![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
 
-### Installation (Not working?)
+### Installation and Usage 
+Note: run all of the following commands in Termux
 
-0. Connect adb to your phone. Run the following command in the Termux window: `adb connect localhost`
-
-1. Clone the repo
-```
-git clone https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor.git && cd WhatsApp-Key-Databse-Extractor
+1. Get dependencies I
+```python
+pkg update && pkg upgrade
+pkg install python git
 ```
-2. Get python requirements
+
+2. Clone the repo
 ```python
-pip install -r requirements.txt
+git clone https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor.git
```

Comment:
&& cd WhatsApp-Key-Databse-Extractor to enter into directory.

Reply:
Happens in the termux_dependencies.py

Doesn't work if you try it with the && , tried it.